src/cpp/rtps/xmlparser/XMLDynamicParser.cpp

```diff
@@ -671,6 +671,13 @@ XMLP_ret XMLParser::parseXMLEnumDynamicType(
      */
     XMLP_ret ret = XMLP_ret::XML_OK;
     const char* enumName = p_root->Attribute(NAME);
+
+    if(enumName == nullptr)
```

Comment:
Uncrustify fails here:
```suggestion
    if (enumName == nullptr)
```